LocalSettings.php

```diff
@@ -108,6 +108,12 @@
 	'wgEnableBlockNoticeStats' => [
 		'default' => false,
 	],
+	'wgEnablePartialBlocks' => [
+		'default' => false,
```

Comment:
Set this to true please and remove the wikis below.

We’re going to enable this for all wikis.